src/gui/main.cpp

```diff
@@ -33,8 +33,8 @@ int ui_main(int argc, char **argv)
 	QCommandLineParser parser;
 	NeovimQt::App::processCliOptions(parser, app.arguments());
 
-	int timeout = parser.value("timeout").toInt();
-	auto c = app.createConnector(parser);
+	int timeout= parser.value("timeout").toInt();
```

Comment:
SpaceBeforeAssignmentOperators: true

Reply:
I dont mind if it ends up changing the style a bit. What I would really like to avoid right now is having to review the entire code base just because of the format :)

Reply:
The goal is definitely to have only `clang-format` induced changes here, and be able to review code styles throughout the code